requirements.txt

```diff
@@ -43,3 +43,8 @@ tzdata==2024.1; python_version >= '2'
 urllib3==2.2.2; python_version >= '3.8'
 werkzeug==3.0.3; python_version >= '3.8'
 zipp==3.20.0; python_version >= '3.8'
+
```

Comment:
Please add the requirements to the Pipfile and not to requirements.txt this file is automatically generated using a script before we deploy the application